fhir-term-graph/kubernetes/umlsTermGraphLoader.yml

```diff
@@ -0,0 +1,37 @@
+kind: Job
```

Comment:
once again, not ideal... should go in different location.

Reply:
Moved.